forge-gui/res/cardsfolder/upcoming/abzan_devotee.txt

```diff
@@ -0,0 +1,7 @@
+Name:Abzan Devotee
+ManaCost:1 B
+Types:Creature Dog Cleric
+PT:2/1
+A:AB$ Mana | Cost$ 1 | Produced$ Combo W B G | ActivationLimit$ 1 | SpellDescription$ Add {W}, {B}, or {G}. Activate only once each turn.
+A:AB$ ChangeZone | Cost$ 2 B | Origin$ Graveyard | Destination$ Hand | ActivationZone$ Graveyard | SpellDescription$ Return this card from your graveyard to your hand.

```

Comment:
@Agetian because of this ability, should this card has `DiscardMe` and `SacMe` flags?

Reply:
Hmm, might be a good idea, yes :+1:

Reply:
I'd say not very high, as the cost (especailly for returning after discard) is high enough, but probably DiscardMe:1 and SacMe:3 or something like that?